HTML/HTML.sublime-syntax

```diff
@@ -15,22 +15,24 @@ variables:
   not_equals_lookahead: (?=\s*[^\s=])
 
   # https://html.spec.whatwg.org/multipage/parsing.html#tag-name-state
-  tag_name: '[[:alpha:]_][^/>\s]*'
+  tag_name_break: (?=[\s/>])
+  tag_name_char: '[^\s/>]'
+  tag_name: '[[:alpha:]_]{{tag_name_char}}*'
```

Comment:
I don't think that underscores are allowed. In addition, I'd spell out :alpha: as `[A-Za-z]` just to make it clear that it's ASCII-only.

Reply:
This originates from my work at XML. You are right. It's not valid as normal tag name in HTML.